src/main/java/com/uber/cadence/internal/sync/SyncActivityWorker.java

```diff
@@ -37,7 +37,9 @@ public SyncActivityWorker(
       IWorkflowService service, String domain, String taskList, SingleWorkerOptions options) {
     taskHandler =
         new POJOActivityTaskHandler(service, domain, options.getDataConverter(), heartbeatExecutor);
-    worker = new ActivityWorker(service, domain, taskList, options, taskHandler);
+    worker =
+        new ActivityWorker(
+            service, domain, taskList, options, taskHandler, "Activity Poller taskList=");
```

Comment:
IMO, `taskList=` part should belong inside ActivityWorker implementation.

Reply:
good catch, agree!